tests/fixtures/bar-foos-bar-route.js

```diff
@@ -0,0 +1,6 @@
+Router.map(function() {
```

Comment:
Can you create an additional fixture that has multiple `bar` routes at the top and nested level? I would expect for them to all be removed, leaving you with `./tests/fixtures/foos-bar-route.js`

```javascript
Router.map(function() {
  this.route('bar');
  this.route('bar');
  this.route('foos', function() {
    this.route('bar');
    this.route('bar');
  });
});
```

---

Can you create an additional fixture that changes the order? I would expect for them to all be removed, leaving you with `./tests/fixtures/foos-bar-route.js`

```javascript
Router.map(function() {
  this.route('foos', function() {
    this.route('bar');
    this.route('bar');
  });
  this.route('bar');
  this.route('bar');
});
```